lib/easybill_rest_client.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
This file should probably stay as `lib/easybill-rest-client.rb` (with a rubocop exception) otherwise gem autoloading won't work and you have to put 

`gem 'easybill-rest-client', require: 'easybill_rest_client'` in there, which is a bit ugly.